concent_api/concent_api/celery.py

```diff
@@ -24,7 +24,15 @@
 for feature_name in settings.CONCENT_FEATURES:
 
     app.conf.imports += tuple(
-        ['{}.tasks'.format(app)
+        [f'{app}.tasks'
          for app in AVAILABLE_CONCENT_FEATURES[feature_name]['required_django_apps']
-         if importlib.util.find_spec('{}.tasks'.format(app)) is not None]
+         if importlib.util.find_spec(f'{app}.tasks') is not None]
     )
+
+app.conf.task_create_missing_queues = True
+app.conf.task_routes = ([
+    ('verifier.tasks.verification_result', {'queue': 'concent'}),
+    ('conductor.tasks.blender_verification_request', {'queue': 'conductor'}),
+    ('verifier.tasks.blender_verification_order', {'queue': 'verifier'}),
+],)
```

Comment:
This has to be tuple ?

Reply:
yes.
http://docs.celeryproject.org/en/latest/userguide/routing.html#automatic-routing (see: _Note_)